src/xenia/app/emulator_window.cc

```diff
@@ -928,6 +936,41 @@ void EmulatorWindow::ToggleFullscreen() {
   SetFullscreen(!window_->IsFullscreen());
 }
 
+void EmulatorWindow::TakeScreenshot() {
+  xe::ui::RawImage myimage;
+  if (emulator()->is_title_open()) {  // Make sure we have a game open first
```

Comment:
The comment it not necessary since it's obvious from the code itself. But, the check is also not strictly required, and everything works without it. For example, if game closing is implemented at some point, maybe we'd want to keep the last frame of the game displayed (and thus saveable), but that's still debatable from the UX point of view.